browserslist

```diff
@@ -1,12 +0,0 @@
-# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
Why is this removed? Is it because angular 10 is not using this anymore? or is it a mistake?